programs/drift/src/instructions/pyth_lazer_oracle.rs

```diff
@@ -0,0 +1,99 @@
+use crate::error::ErrorCode;
+use crate::state::pyth_lazer_oracle::{
+    PythLazerOracle, PYTH_LAZER_ORACLE_SEED, PYTH_LAZER_STORAGE_ID,
+};
+use crate::validate;
+use anchor_lang::prelude::*;
+use pyth_lazer_sdk::protocol::payload::{PayloadData, PayloadPropertyValue};
+use solana_program::sysvar::instructions::load_current_index_checked;
+
+pub fn handle_update_pyth_lazer_oracle(
+    ctx: Context<UpdatePythLazerOracle>,
+    feed_id: u32,
+    pyth_message: Vec<u8>,
+) -> Result<()> {
+    let ix_idx = load_current_index_checked(&ctx.accounts.ix_sysvar.to_account_info())?;
+    validate!(
+        ix_idx > 0,
+        ErrorCode::InvalidVerificationIxIndex,
+        "instruction index must be greater than 0 to include the sig verify ix"
+    )?;
+
+    let verified = pyth_lazer_sdk::verify_message(
```

Comment:
is this only think we use from pyth_lazer_sdk? can we just pull this into our own package so we dont needa take a dependency?

Reply:
i'd like to leave it for now becuase they said it's still in beta and that their code/api can change. once it's finalized i agree